src/cascade_at/dismod/api/run_dismod.py

```diff
@@ -24,3 +24,26 @@ def run_dismod(dm_file, command):
     info.stderr = process.stderr.decode()
     
     return info
+
+
+def run_dismod_commands(dm_file, commands):
+    """
+    Runs multiple commands on a dismod file and returns the exit statuses.
+    Will raise an exception if it runs into an error.
+
+    Args:
+        dm_file:
+        commands:
+
+    Returns:
+
+    """
+    for c in commands:
+        process = run_dismod(dm_file=dm_file, command=c)
+        if process.exit_status:
+            LOG.error(f"{c} failed with exit_status {process.exit_status}:")
```

Comment:
# I wonder if it might be worthwhile to print stdout too, go give a person some context for finding what caused the error.